### PR TITLE
Add SES service support

### PR DESCRIPTION
**Pull request by lessless**
Hello folks,

Please, welcome SES service support. This PR introduce a very modest piece of functionality but it's just to kick-off the things.

### CODE REVIEW

File: test/lib/ex_aws/ses_test.exs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  test "#identity_verification_attributes", ctx do
+    expected = %{
+      "Action" => "GetIdentityVerificationAttributes",
+      "Identities.member.1" => ctx.email
+    }
+
+    assert expected = SES.identity_verification_attributes([ctx.email]).params
+  end
+end
```
Comment (benwilson512) [inline test/lib/ex_aws/ses_test.exs]: You need to include a test that actually hits SES, usually a basic list operation.

Comment (lessless) [inline test/lib/ex_aws/ses_test.exs]: Do you mean something like ` ExAws.SES.configuration_sets() |> ExAws.request()` ?

**Comment by lessless** [comment]
@benwilson512  I fixed endpoint location, added `SendEmail` API call and a test with real request.
Please review again.

**Comment by benwilson512** [comment]
Hey! So there's still a few tweaks I'd like to make, but we also need SES support now. So my plan is to merge this into an `ses-dev` branch so that I can make changes. I'll ping you on that PR.

**Comment by lessless** [comment]
@benwilson512 looking forward to!

**Pull request merged**